lib/util.js

```diff
@@ -89,13 +106,23 @@ function isFileProtocol(url) {
 /**
  * Check if value is a valid number
  * 
- * @param {Number} value Check if value is a valid number
+ * @param {Number} value Value to check
  * @returns {Boolean} True if value is a valid number
  */
 function isValidNumber(value) {
     return (typeof value === "number") && !isNaN(value);
```

Comment:
do you want/need to check if its finite? (`!isInfinity()`)

Reply:
Good catch, thanks @jdelbick !

Reply:
It looks like `Number.isFinite` can change the entire function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite